gdal/port/cpl_odbc.h

```diff
@@ -159,7 +159,7 @@ class CPLODBCStatement;
  */
 
 class CPL_DLL CPLODBCSession {
-    char      m_szLastError[SQL_MAX_MESSAGE_LENGTH + 1];
+    CPLString m_szLastError;
```

Comment:
The variable should be renamed m_osLastError then

Reply:
You're right, will correct it.